module/VuFind/src/VuFind/Content/Excerpts/Syndetics.php

```diff
@@ -118,13 +118,11 @@ public function loadByIsbn($key, \VuFindCode\ISBN $isbnObj)
 
                 // Get the marc field for copyright (997)
                 $nodes = $xmldoc->GetElementsbyTagName('Fld997');
-                if ($nodes->length) {
-                    $excerpt[$i]['Copyright'] = html_entity_decode(
-                        $xmldoc2->saveXML($nodes->item(0))
-                    );
-                } else {
-                    $excerpt[$i]['Copyright'] = null;
-                }
+                $excerpt[$i]['Copyright'] = $nodes->length
+                ? html_entity_decode(
+                    $xmldoc2->saveXML($nodes->item(0))
+                )
+                : null;
```

Comment:
This could use an indent and unwrap:

```suggestion
                    ? html_entity_decode($xmldoc2->saveXML($nodes->item(0)))
                    : null;
```